lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -73,6 +73,23 @@ set_input_value:
       xpath: //input[@id='<input_field_id>']
     op: send_keys <input_value>
     type: input
+click_first_item_from_resource_list_and_check_breadcrumb:
+  action: click_first_item_in_resource_list
+  action: check_link_in_breadcrumb
+click_first_item_from_sub_resource_list_and_check_breadcrumb:
+  action: click_first_sub_item_in_resource_detail_list
+  action: check_link_in_breadcrumb
+click_first_item_from_gridcell_list_and_check_breadcrumb:
+  action: click_first_item_in_grid_cell_list
+  action: check_link_in_breadcrumb
```

Comment:
Update the rules named as `click first item from xxx and check xxx`.
Because it combines the `PURE click` action and `PURE check` action.
The reason to combine click and check together in one rule is to avoid URL change or listed resource layout change.
1. `click_first_item_from_resource_list_and_check_breadcrumb` is click the first item from resource list such as `projects`, `pods`, etc.
2. `click_first_item_from_sub_resource_list_and_check_breadcrumb:` is click the first sub item from a resource such as one of `imagestreamtag` on `imagestream` detail page
3. `click_first_item_from_gridcell_list_and_check_breadcrumb:` is to click the first item from resources listed in grid cell, such as `installed operator`